lib/hanami/assets/compilers/sass.rb

```diff
@@ -31,14 +31,12 @@ def renderer
 
         # @since 0.3.0
         # @api private
-        def dependencies
-          engine.dependencies.map { |d| d.options[:filename] }
+        def engine
+          ::SassC::Engine.new(to_be_compiled, load_paths: load_paths, cache_location: CACHE_LOCATION)
         end
 
-        # @since 0.3.0
-        # @api private
-        def engine
-          ::Sass::Engine.for_file(source.to_s, load_paths: load_paths, cache_location: CACHE_LOCATION)
+        def to_be_compiled
```

Comment:
```suggestion
        # @since x.x.x
        # @api private
        def to_be_compiled
```

Reply:
@landongrindheim Please commit this suggestion.